jaeger-core/src/main/java/com/uber/jaeger/samplers/ProbabilisticSampler.java

```diff
@@ -33,25 +33,38 @@
 public class ProbabilisticSampler implements Sampler {
   public static final String TYPE = "probabilistic";
 
-  private final long positiveSamplingBoundary;
-  private final long negativeSamplingBoundary;
-  private final double samplingRate;
-  private final Map<String, Object> tags;
+  private final double DELTA = 0.00001;
+  private long positiveSamplingBoundary;
+  private long negativeSamplingBoundary;
+  private double samplingRate;
+  private Map<String, Object> tags;
```

Comment:
I am concerned about garbage collection when a service has many endpoints; updating these in place would perform much better than creating a new object on each update for every endpoint.